lib/desired-caps.js

```diff
@@ -248,6 +248,12 @@ let commonCapConstraints = {
   },
   mockLocationApp: {
     isString: true
+  },
+  logcatFormat: {
+    isArray: true,
```

Comment:
Should here be isString?

Reply:
[it is](https://github.com/appium/appium-android-driver/commit/82f25257df4eb48814e1b4f878f9c4ede6beb7b8) ;)

Reply:
ah, yey 👍